app/src/pages/api/business/schema.graphql

```diff
@@ -8,8 +8,31 @@ type Business {
 type BusinessCampaign {
   btcPayServerStoreId: String
   businessId: String
+  content: BusinessCampaignContent
+  expiresAt: Date
   id: String!
+  investmentMultiple: Float
   slug: String
+  totalSatsInvested: Float
+}
+
+type BusinessCampaignContent {
+  asHtmlString: String
+  category: String
+  country: String
+  description: String
+  establishedAt: String
+  instagram: String
+  latitude: String
+  longitude: String
+  markerIcon: String
+  media: BusinessCampaignContentMedia
+  title: String
+  website: String
+}
```

Comment:
Let's make all these props mandatory. 
This will enforce the Content Management team to complete all page custom fields of a given business campaign. 

Another benefit of enforcing properties is that in the component, we don't need to cast them as not `undefined`